### DISCUSSION TITLE
zabbix query

### DISCUSSION THREAD

**Discussion by yosiasz** [grafana/grafana-infinity-datasource#701] [Q&A]
Hello

how would one go about using the `--data` parameter in infinity plugin?

```
curl --request POST \
  --url 'https://example.com/zabbix/api_jsonrpc.php' \
  --header 'Content-Type: application/json-rpc' \
  --data '{"jsonrpc":"2.0","method":"host.get","params":{"output":["hostid"]},"auth":"0424bd59b807674191e7d77572075f33","id":1}'
  ```
  
  Thanks

**Comment by yosiasz** [comment]
sorted out by adding Bearer to the value

<img width="202" alt="image" src="https://github.com/yesoreyeram/grafana-infinity-datasource/assets/900951/8bfe7782-a49c-47d6-bc47-4e70531489f3">

**Reply by yesoreyeram** [reply]
BTW, it is strongly recommend not to use any secure information in the query. They should only exist in the config.

**Reply by yosiasz** [reply]
Thanks @yesoreyeram indeed but I was unable to find a url field in the the datasource config side of infinity. Let me poke around some more

**Reply by yosiasz** [reply]
I do not see URL in this config of infinity
![image](https://github.com/yesoreyeram/grafana-infinity-datasource/assets/900951/7fe4e362-4712-4255-89d2-ac74f8703487)
